test_scripts/API/Restructuring_OnResetTimeout/021_BC_OnResetTimeout_for_GetInteriorVehicleDataConsent_without_SetInteriorVehicleData_response.lua

```diff
@@ -0,0 +1,80 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check SDL postpones response to a specific Remote Control RPCs until user consent is received
+--  and respond with GENERIC_ERROR:false to Mobile app in case HMI hasn't responded
+-- Applicable RPCs: 'SetInteriorVehicleData', 'ButtonPress'
+------------------------------------------------------------------------------------------------------------------------
+-- Preconditions:
+-- 1) Default SDL timeout is 10s (defined in .INI by 'DefaultTimeout' parameter)
+--
+-- In case:
+-- 1) RC access mode is set to 'ASK_DRIVER'
+-- 2) RC <module> is allocated by mobile App_1
+-- 3) App_2 sends 'SetInteriorVehicleData' RPC for the RC <module>
+-- 4) SDL sends 'RC.GetInteriorVehicleDataConsent' to HMI
+-- 5) HMI sends 'BC.OnResetTimeout' to SDL with 'resetPeriod=25s' for 'RC.GetInteriorVehicleDataConsent'
+--  right after receiving request
+-- 6) HMI responds to 'RC.GetInteriorVehicleDataConsent(allow)' with SUCCESS resultCode in 23s
+-- 7) SDL sends 'RC.SetInteriorVehicleData' request to HMI
+-- 8) HMI does not respond to 'RC.SetInteriorVehicleData' request
```

Comment:
So this is second scenario where SDL sends a new request to HMI and HMI does not respond at all with either `onResetTimeOut` or the actual response, correct?

Reply:
@atiwari9 Please find a sequence diagram for this scenario: 
<img src="http://www.plantuml.com/plantuml/svg/PP11ImCn48Nl-HLpt1uYMl6owDAIBdL1BIJMguJk70hRP4bcrPzlPeSYYyUycVVcozjayeccow2suwWRjtkfmI5r7xjgfSfBDy-lNGtMtEndrSIGCP0ggcwE6PDI_aJdBqy8yfJH77G85ZFI-_c2SQ9g_P1NPSvhugrXVGyPJoB9fUqmn6_LupALCwzqHsDQv_upLPwjs3zHcf0FBVdbLt-qHHTMsRYxPHAsmC-ZdgKG8M4UERq0BOoSh9jIMGtxzgsrdVbihJtOQtbgWw4lhVu2">